rio/subsystems/elevator/elevator_constants.py

```diff
@@ -73,4 +81,16 @@ class Elevator_Constants:
     kElevatorPositionKey = "ElevatorPosition"
 
     kElevatorManualChange = 0.01
-    """.01 meters / (1/50) seconds = 0.5 m/s"""
\ No newline at end of file
+    """.01 meters / (1/50) seconds = 0.5 m/s"""
+
+    kManualUpDutyCycle = 0.2
+    """ Power going up manually"""
+
+    kManualDownDutyCycle = -0.2
+    """ Power going down manually"""
+
+    kManualNoPower = 0
+    """ Turn off motors """
+
+    CARRIAGE_HEIGHT_AT_BOTTOM_M = units.inchesToMeters(8.0) # TODO measure
```

Comment:
Would be nice to have a better name for this, not sure what it should be

Reply:
I'm assuming this is just the carriage height when the robot is in-taking coral from the substation. 
```suggestion
    CARRIAGE_HEIGHT_AT_INTAKING = units.inchesToMeters(8.0) # TODO measure
```

Reply:
Actually, it is when the carriage is at the bottom in meters. It is likely that at the bottom and intaking are the same, but if for some reason we end up needing to be slightly off the bottom for intaking, this value would NOT change. However, the height value for the Floor.INTAKING enum would change.

TLDR: This really is bottom, not intaking.

Reply:
Perhaps renaming Floor.START to Floor.BOTTOM would make sense.